src/lib/structures/game/RoundData.ts

```diff
@@ -0,0 +1,16 @@
+import type { Snowflake } from 'discord.js';
+import { cleanSongName, cleanArtistName } from '#utils/audio';
+import { Collection } from 'discord.js';
+
+export class RoundData {
+	public validSongVariations: string[] = [];
+	public guessedSong: Snowflake[] = [];
+	public guessedArtists = new Collection<string, Snowflake[]>();
+	public primaryArtist = '';
+
+	public constructor(song: string, artists: string[]) {
+		this.validSongVariations = cleanSongName(song);
+		this.guessedArtists = new Collection(artists.map((artist) => [cleanArtistName(artist), []]));
+		this.primaryArtist = this.guessedArtists.firstKey() ?? this.primaryArtist;
+	}
+}
```

Comment:
```suggestion
	}
	
	public get primaryArtistGuessers() {
		return this.guessedArtists.get(this.primaryArtist)!;
	}
}
```

Alternatively, you can just do `this.guessedArtists.first()!`. Alternative alternatively, omit a getter and replace all use of `round.guessedArtists.get(round.primaryArtist)!` with `round.guessedArtists.first()!` in the code.